src/utils/time-fmt.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * Format the time as hh:mm:ss, dropping the hour part if it is zero.
+ * @param seconds - the number of seconds in the duration
+ * @returns the duration in a human-friendly format
+ */
+export const timeFmt = (seconds: number): string => {
+  const date = new Date(0)
+  date.setSeconds(Number.isFinite(seconds) ? seconds : 0)
+  return date.toISOString().substr(11, 8).replace(/^00:/, '')
```

Comment:
Just noting that this will fail for audio files longer than 24 hours (as it'll just wrap back around to 0 at that point).

It'd be nice if seconds and minutes weren't base _whatever they are_.

Here's a solution on SO that is to the point and doesn't have any problems with very, very long audio:

https://stackoverflow.com/a/11486026

Reply:
Fixed in 1ae858d with a slightly tweaked implementation of the SO answer.